### ISSUE TITLE
get user email from twitter

### ISSUE THREAD

**Issue by ahuggins** [atymic/twitter#154]
After quite a bit of digging, I tracked down how to get the user email from Twitter. Basically, when you request the credentials, you need to pass an array like this:

```
$credentials = Twitter::getCredentials([
     'include_email' => "true",
]);
```

The key thing to note is that I am passing `true` as a string, and not as a boolean value. This is why `'include_email' => 1` does not work either. When the request is made to Twitter, `true` needs to be in the GET parameter...if you use the boolean value, it is converted eventually to 1.

It might be worth mentioning this in the docs somewhere. I might make a pull request, but wanted to make a note here as a quick action.

this might be the same issue as these:
https://github.com/thujohn/twitter/issues/150
https://github.com/thujohn/twitter/issues/134

**Comment by ahuggins** [comment]
I just submitted a pull request to update the docs with the relevant information.

**Comment by jagomez8** [comment]
Thank you @ahuggins! This saved me undoubtedly a few hours of pulling my hair out.

Another thing to note is that your app won't retrieve emails by default. you have to enable it in your app's permissions page.

<img width="960" alt="screen shot 2016-09-29 at 11 14 58 am" src="https://cloud.githubusercontent.com/assets/1369937/18966497/3c8a0774-8636-11e6-8e1b-de0ebe0c660e.png">

**Comment by ahuggins** [comment]
Yeah I noted that in the Pull Request I made, but it hasn't been merged yet. Glad you found it helpful!

**Comment by atymic** [comment]
Closed in #155
